pmm/pmm2-server-autobuild.groovy

```diff
@@ -30,6 +30,9 @@ pipeline {
 
                 git poll: true, branch: GIT_BRANCH, url: 'http://github.com/Percona-Lab/pmm-submodules'
                 sh '''
+                    set -o errexit
+                    set -o xtrace
```

Comment:
when you add `set -o xtrace` please make sure that you do not show any secrets

Reply:
Well, yeah – only on that line. Do we need other `xtrace` lines added in that PR? Is it safe?